components/neverbounce/app/neverbounce.app.ts

```diff
@@ -0,0 +1,40 @@
+import { defineApp } from "@pipedream/types";
+import { axios } from "@pipedream/platform";
+import {
+  HttpRequestParams,
+  VerifyEmailParams, VerifyEmailResponse,
```

Comment:
Can we put them all to the same line or can we have them all in different lines?

Reply:
I thought this was standardized with eslint but apparently it does not change this line.

I put them all on the same line. Thanks for noting!